eng/pipelines/templates/steps/cmake-build.yml

```diff
@@ -15,6 +15,19 @@ steps:
     workingDirectory: build
     displayName: cmake --version
 
+  - script: vcpkg --version
+    workingDirectory: build
+    condition: >-
+      and(
+        succeeded(),
+        not(contains(variables['OSVmImage'], 'macOS'))
+      )
+    displayName: vcpkg --version
```

Comment:
Do we fetch vckpg down during build? https://github.com/Azure/azure-sdk-for-cpp/blob/main/cmake-modules/AzureVcpkg.cmake#L28-L32

If so, the vcpkg on `PATH` might not be the one used during the build.

Reply:
moved to vcpkg clone thingy